src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -11310,6 +11310,9 @@ void sofia_handle_sip_i_invite(switch_core_session_t *session, nua_t *nua, sofia
 				} else if (profile->paid_type == PAID_VERBATIM) {
 					switch_channel_set_variable(channel, "sip_P-Asserted-Identity",  full_paid_header);
 				}
+				
+				// We want the full PAID to be recorded without risk of transfering the data to other call legs
+				switch_channel_set_variable(channel, "p_aid_full",  full_paid_header);
```

Comment:
I think `full_paid_header` is a good name instead of `p_aid_full`

Reply:
While that may slightly break convention, I chose `p_aid_full` instead of `full_paid_header` because 'paid' could mean "p-asserted-identity' or it could mean 'paid as in paid the bill'.

Reply:
@rogelio-telnyx requested the change as well.  Done.